CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 0.1.37
+
+### Enhancements
+
+- Change default HTTP read timeout to 30s ([#3182](https://github.com/astral-sh/uv/pull/3182))
```

Comment:
This actually changed with the original http timeout -> read timeout PR, but it's good to make this clear here.

```suggestion
- Change default HTTP read timeout to 30s. Note that uv does not have a timeout for the entire HTTP request, but only between two reads in a request ([#3182](https://github.com/astral-sh/uv/pull/3182))
```

Reply:
Sorry didn't see this before the auto-merge. I'll make this note in the GitHub release.